python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -254,6 +260,10 @@ def execute_asset_backfill_iteration(
     if result.backfill_data.is_complete():
         updated_backfill = updated_backfill.with_status(BulkActionStatus.COMPLETED)
 
+    pipeline_and_execution_plan_cache: Dict[
```

Comment:
Idea here that you can choose to ignore:

```python
class PipelineAndExecutionPlanCache:
    def __init__(workspace):
        ...

    @cached_method
    def get_pipeline_and_execution_plan(self, selector):
        code_location = workspace.get_code_location(repo_handle.code_location_origin.location_name)
        ...
```

Reply:
not ideologically opposed but the specific I am choosing to ignore is that I don't think we can pass the workspace into the constructor - since it may change between calls after https://github.com/dagster-io/dagster/pull/13085